### ISSUE TITLE
triggers: Transaction spans entire procedure

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#358]
We have database write / create / delete triggers for logging data - the procedure for each is a .p - each of these is being flagged by the "Transaction spans entire procedure" rule, which I do not think makes much sense within the scope of a TRIGGER PROCEDURE

````
TRIGGER PROCEDURE FOR CREATE OF customer.

ASSIGN
   customer.name = "changed"
   .
```` 

I am not sure if the above is sufficient to trigger the rule - we are doing more.

**Comment by gquerret** [comment]
This has been fixed a few days ago, see issue #24

**Comment by gquerret** [comment]
Please note that it's not yet included in a released version
